.github/workflows/smoketests.yml

```
@@ -0,0 +1,79 @@
+name: Run Smoke Tests - Deriv app
+
+permissions:
+    actions: write
+    checks: write
+    contents: write
+    deployments: write
+    pull-requests: write
+    statuses: write
+
+on: 
+    issue_comment:
+        types: [edited]
+        
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: Runner_16cores_Deriv-app
```
since we have this runner on deriv-app we can use this.